### PR TITLE
govee2mqtt: init at 2024.01.21-088d4ca8, nixos/govee2mqtt: init

### PR DESCRIPTION
**Pull request by SuperSandro2000**
## Description of changes

Deployed and configured locally and works as expected with my lights.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/go/govee2mqtt/package.nix [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  buildInputs = [ openssl ];
+
+  postInstall = ''
+    mkdir -p $out/share/govee2mqtt/
+    cp -r assets $out/share/govee2mqtt/
+  '';
+
+  meta = with lib; {
+    description = "Govee2MQTT: Connect Govee lights and devices to Home Assistant";
```
Comment (AndersonTorres) [inline pkgs/by-name/go/govee2mqtt/package.nix:49]:
```suggestion
    description = "Connect Govee lights and devices to Home Assistant";
```

Self-reference not allowed.

File: pkgs/by-name/go/govee2mqtt/package.nix [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    homepage = "https://github.com/wez/govee2mqtt";
+    changelog = "https://github.com/wez/govee2mqtt/blob/main/addon/CHANGELOG.md";
```
Comment (AndersonTorres) [inline pkgs/by-name/go/govee2mqtt/package.nix:51]:
```suggestion
    changelog = "https://github.com/wez/govee2mqtt/blob/${src.rev}/addon/CHANGELOG.md";
```

**Review by AndersonTorres — COMMENTED** [review]
LGTM.

Result of `nixpkgs-review pr 285298` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>govee2mqtt</li>
  </ul>
</details>

P.S.: is this expected?

```text
[nix-shell:~/.cache/nixpkgs-review/pr-285298]$ ./results/govee2mqtt/bin/govee -V
govee 0.1.0
```

**Review by pbsds — APPROVED** [review]
forgot to submit yesterdays review.

The nixos manual and package build outputs LGTM. Didn't test the module, but the code LGTM.

**Comment by SuperSandro2000** [comment]
> Didn't test the module

I did that with physical hardware and it works better than the previous home-assistant plugins after configuring correct.

**Pull request merged**